src/image-upload/image-upload.component.ts

```diff
@@ -26,6 +26,7 @@ export class ImageUploadComponent implements OnInit {
 
   @Input() beforeUpload: (UploadMetadata) => UploadMetadata | Promise<UploadMetadata> = data => data;
   @Input() buttonCaption: string = 'Select Images';
+  @Input('class') klass: string = 'img-ul';
```

Comment:
Personally I don't like the name `klass`, it doesn't really tell me what the variable is - perhaps `cssClass` or `customClass` fit better.